beancount/query/query_compile_test.py

```diff
@@ -171,28 +171,28 @@ def test_compile_EvalOr(self):
         self.assertEqual(bool, c_or.dtype)
 
     def test_compile_EvalMul(self):
-        c_plus = qc.EvalMul(qc.EvalConstant(17), qc.EvalConstant(18))
+        c_plus = qc.EvalMul(qc.EvalConstant(D(17)), qc.EvalConstant(D(18)))
         self.assertEqual(Decimal, c_plus.dtype)
 
     def test_compile_EvalDiv(self):
-        c_plus = qc.EvalDiv(qc.EvalConstant(17), qc.EvalConstant(18))
+        c_plus = qc.EvalDiv(qc.EvalConstant(D(17)), qc.EvalConstant(D(18)))
         self.assertEqual(Decimal, c_plus.dtype)
 
     def test_compile_EvalAdd(self):
-        c_plus = qc.EvalAdd(qc.EvalConstant(17), qc.EvalConstant(18))
+        c_plus = qc.EvalAdd(qc.EvalConstant(D(17)), qc.EvalConstant(D(18)))
         self.assertEqual(Decimal, c_plus.dtype)
 
     def test_compile_EvalSub(self):
-        c_plus = qc.EvalSub(qc.EvalConstant(17), qc.EvalConstant(18))
+        c_plus = qc.EvalSub(qc.EvalConstant(D(17)), qc.EvalConstant(D(18)))
         self.assertEqual(Decimal, c_plus.dtype)
 
     def test_compile_EvalAddAmount(self):
-        c_plus = qc.EvalAdd(qc.EvalAmount(Decimal('1.11','USD')), qc.EvalAmount(Decimal('3.55','USD')))
-        self.assertEqual(Decimal, c_plus.dtype)
+        c_plus = qc.EvalAdd(qc.EvalConstant(amount.Amount(Decimal('1.11'),'USD')), qc.EvalConstant(amount.Amount(Decimal('3.55'),'USD')))
```

Comment:
`D('1.11')` instead of `Decimal('1.11')` for consistency with the other tests. Or you could also add a `from beancount.core.amount import A` and simply write `A('1.11 USD')` for brevity.

Reply:
Done